echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/PipelineConfigsPollingAgent.groovy

```diff
@@ -176,7 +182,7 @@ class PipelineConfigsPollingAgent extends AbstractPollingAgent {
        */
       List<Pipeline> pipelinesWithNewTriggers = pipelines.findAll {
         def cronTriggers = it.triggers.findAll { it.type == Trigger.Type.CRON.toString() }
-        cronTriggers.any { !actionIds.contains(it.id) }
+        cronTriggers.any { !actionIds.contains(it.id) && !actionIds.find { actionId -> actionId.endsWith(":$it.id") } }
```

Comment:
This "endsWith" check could probably be factored out. In fact, the whole id matching probably should be to avoid accidentally writing code that compares IDs incorrectly.